service_resource_test.go

```diff
@@ -0,0 +1,140 @@
+package twilio
+
+import (
+	"context"
+	"testing"
+)
+
+func TestFetchMessagingResource(t *testing.T) {
+	t.Parallel()
+	client, server := getServer(messagingResourceResponse)
+	defer server.Close()
+	msr, err := client.Resource.ServiceResourceService.Fetch(context.Background(), "MGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
```

Comment:
Could we change this to `client.Messaging.ServiceResources.Fetch` or similar?

Reply:
Unfortunately `Messaging` field exists in `Client` structure, so I changed `ServiceResourceService` to just `ServiceResources`.
I understand that `Resource` name is very generic one but I could not fine suitable name.

Maybe we can use `Copilot` name as it is main feature of the Messaging services. What do you think?